jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    testPathIgnorePatterns: ["/.next/", "/node_modules/"],
+    setupFilesAfterEnv: ["./setupTests.js"],
```

Comment:
Can you move this into `test/lib/setup.js` to keep all tests related code under `test/`? The `jest.Timeout` directive in that file currently does not have any impact, so you can remove it. It was added long back when tests were running in dev environment.